weaver/wps_restapi/swagger_definitions.py

```diff
@@ -730,7 +745,8 @@ class LinkRelationshipType(OneOfKeywordSchema):
             "Relationship of the link to the current content. "
             "This should be one item amongst registered relations https://www.iana.org/assignments/link-relations/."
         )),
-        URL(description="Fully qualified extension link relation to the current content.")
+        URL(description="Fully qualified extension link relation to the current content."),
+        NRST(description="Link to Job output_id.") # NamespacedRelationshipType
```

Comment:
Replace by something along the lines of "Namespaced relationship type.". 
Even if used in this case for job outputs, it is not exclusive to it in this schema.